### ISSUE TITLE
Difficulty running singularity image of 0.15.2:  Read-only file system: '/home/bidsapp/.cache/mriqc'

### ISSUE THREAD

**Issue by yarikoptic** [poldracklab/mriqc#853]
I am not sure if it is a problem in mriqc docker image itself, or may be it is due to the difference in singularity version used to produce 0.15.1 and 0.15.2 images (from docker images) for https://github.com/ReproNim/containers ... but with 0.15.2, if I follow our https://github.com/ReproNim/containers#a-typical-workflow (just remove/comment out the freezing to 0.15.1 so you would get 0.15.2) I would get

```shell
> datalad containers-run -n containers/bids-mriqc --input sourcedata --output . '{inputs}' '{outputs}' participant group
[INFO   ] Making sure inputs are available (this may take some time)
get(ok): /home/yoh/.tmp/repro-KtxiqAt/ds000003-qc/sourcedata/sub-02/anat/sub-02_T1w.nii.gz (file) [from s3-PUBLIC...]
get(ok): /home/yoh/.tmp/repro-KtxiqAt/ds000003-qc/sourcedata/sub-02/anat/sub-02_inplaneT2.nii.gz (file) [from s3-PUBLIC...]
get(ok): /home/yoh/.tmp/repro-KtxiqAt/ds000003-qc/sourcedata/sub-02/func/sub-02_task-rhymejudgment_bold.nii.gz (file) [from s3-PUBLIC...]
get(ok): /home/yoh/.tmp/repro-KtxiqAt/ds000003-qc/sourcedata/sub-13/anat/sub-13_T1w.nii.gz (file) [from s3-PUBLIC...]
get(ok): /home/yoh/.tmp/repro-KtxiqAt/ds000003-qc/sourcedata/sub-13/anat/sub-13_inplaneT2.nii.gz (file) [from s3-PUBLIC...]
get(ok): /home/yoh/.tmp/repro-KtxiqAt/ds000003-qc/sourcedata/sub-13/func/sub-13_task-rhymejudgment_bold.nii.gz (file) [from s3-PUBLIC...]
get(ok): /home/yoh/.tmp/repro-KtxiqAt/ds000003-qc/containers/images/bids/bids-mriqc--0.15.2.sing (file) [from origin...]
[INFO   ] == Command start (output follows) =====
Traceback (most recent call last):
  File "/usr/local/miniconda/bin/mriqc", line 10, in <module>
    sys.exit(main())
  File "/usr/local/miniconda/lib/python3.7/site-packages/mriqc/cli/run.py", line 15, in main
    parse_args()
  File "/usr/local/miniconda/lib/python3.7/site-packages/mriqc/cli/parser.py", line 368, in parse_args
    parser = _build_parser()
  File "/usr/local/miniconda/lib/python3.7/site-packages/mriqc/cli/parser.py", line 340, in _build_parser
    latest = check_latest()
  File "/usr/local/miniconda/lib/python3.7/site-packages/mriqc/cli/version.py", line 22, in check_latest
    cachefile.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/local/miniconda/lib/python3.7/pathlib.py", line 1241, in mkdir
    self._accessor.mkdir(self, mode)
OSError: [Errno 30] Read-only file system: '/home/bidsapp/.cache/mriqc'
[INFO   ] == Command exit (modification check follows) =====
[INFO   ] The command had a non-zero exit code. If this is expected, you can save the changes with 'datalad save -d . -r -F .git/COMMIT_EDITMSG'
CommandError: ''containers/scripts/singularity_cmd run containers/images/bids/bids-mriqc--0.15.2.sing '"'"'sourcedata'"'"' '"'"''"'"' participant group'' failed with exitcode 1 under /home/yoh/.tmp/repro-KtxiqAt/ds000003-qc
```

I have not really figured out **why** 0.15.2 specifically falls into the trap, and how `~/` inside the container is not the one we prepare/provide to singularity invocation (in [here](https://github.com/ReproNim/containers/blob/master/scripts/singularity_cmd#L82)) or is it the singularity version (3.5.2+ds1-1~ndall+1) to matter.

May be there is some difference in version of singularity which was used to build those two versions awhile back, but their recipes are nearly identical:
```
(git)smaug:~/proj/repronim/containers[master]git
$> diff images/bids/Singularity.bids-mriqc--0.15.{1,2}
6c6
< From: poldracklab/mriqc:0.15.1
---
> From: poldracklab/mriqc:0.15.2
15,16c15,16
< export LANG="en_US.UTF-8"
< export LC_ALL="en_US.UTF-8"
---
> export LANG="C.UTF-8"
> export LC_ALL="C.UTF-8"
```

So just decided to reach out may be for bright ideas/memories of possible changes between 0.15.1 and 0.15.2 which could attribute here (I will test later more by rebuilding both singularity images with consistent/same version etc)

Relates in spirit to an elderly #709 ;-)

**Comment by yarikoptic** [comment]
Great to have github - I can find the issues I have filed awhile back and forgot about.  But memory gets better when I keep running into the ones which aren't resolved ;)

**Comment by yarikoptic** [comment]
attn @effigies

**Comment by effigies** [comment]
Looks like we had to fix this on fMRIPrep, as well, so I just copied the fix in #862.
